### ISSUE TITLE
JSON schema/example has describesPackages/documentDescribes property that has no equivalent in the spec

### ISSUE THREAD

**Issue by SamuraiAku** [spdx/spdx-spec#533]
The schema has a property called describesPackages at the top level

```
"describesPackages" : {
      "description" : "The describesPackage property relates an SpdxDocument to the package which it describes.",
      "type" : "array",
      "items" : {
        "description" : "SPDX ID for Package.  The describesPackage property relates an SpdxDocument to the package which it describes.",
        "type" : "string"
      }
```

In the JSON example, there is instead a property called documentDescribes which looks like it is the same thing:
`"documentDescribes" : [ "SPDXRef-File", "SPDXRef-Package" ],`

The issue is that I can't find anything in Section 2 of the specification that corresponds to this.  The description above isn't all that clear to me either but it sounds to me like it's describing a relationship (section 7).  But that already has its own sub-section in the schema.

In section 7 of the spec it says (not very obviously) that if there is more than one package or set of files, then a single relationship is mandatory which describes the relationship between the document and the top-level package. Which is what this extra property is doing.

I would recommend that "describesPackages" be deleted from the schema and "documentDescribes" from the example and use relationships instead per section 7 of the spec.  Anything else is just going to be confusing to a newcomer trying to make the properties correspond to the spec and when trying to translate between JSON and another format such as tag-value.

**Comment by goneall** [comment]
@SamuraiAku Another good catch - the schema should be using "documentDescribes" based on previous discussions in issue #115   I'll update the PR to fix this.

There is general agreement in the SPDX tech team that this is confusing having both relationships and a documentDescribes - this is an issue for the tag/value and YAML formats in addition to the JSON format.  The current proposed solution would create an incompatibility, so we are planning on fixing this in SPDX 3.0 which allows for changes that are not backwards compatible.

BTW - if you haven't already, you may want to sign up for the SPDX tech mailing list or join the tech calls if you are interested in further discussions on the spec.  Information can be found here: https://spdx.dev/participate/tech/

**Comment by goneall** [comment]
Just a note - in looking for documentation on `documentDescribes`, there really isn't anything published in 2.2 making this very confusing as noted by @SamuraiAku  

There are a few questions we need to answer to properly document and remove some of the confusion:
- What is the description for this field?
- What is the cardinality (e.g. is it required)?
- When converting to other formats, do we changes these to DOCUMENT_DESCRIBES relationships? (I think the answer is yes)
- When converting from other formats to YAML and JSON do we generate the `documentDescribes` array from the relationships (I think the answer is yes)
- Do we retain the document relationships DOCUMENT_DESCRIBES or remove them when converting to JSON and YAML?

@tsteenbe - I know we discussed some of these previously - let me know your thoughts on the above.

An alternative to deciding and documenting would be to remove the documentDescribes property as suggested by @SamuraiAku , but I'm reluctant to re-open the discussion after we have already made a decision.
